### DISCUSSION TITLE
Webpack in NODE_ENV=development question

### DISCUSSION THREAD

**Discussion by james0r** [uicrooks/shopify-theme-lab#126] [Q&A]
What kind of ES6 tranpilation is happening when run in development mode? I noticed when I deployed without using @babel/plugin-transform-runtime my async functions were failing, but they were working in development.

**Comment by sergejcodes** [comment]
There is no transpilation happening in development. If you are using async functions with babel (loader included by default for production) then you need @babel/plugin-transform-runtime otherwise they don't work.
